src/main/java/resources/init_db.sql

```diff
@@ -1,9 +1,143 @@
+CREATE TABLE `internet_shop`.`users` (
+  `user_id` BIGINT(11) BIGINT NOT NULL AUTO_INCREMENT,
+  `login` VARCHAR(256) NOT NULL,
+  `username` VARCHAR(256) NOT NULL,
+  `password` VARCHAR(256) NOT NULL,
+  `deleted` TINYINT NULL DEFAULT 0,
+  PRIMARY KEY (`user_id`),
+  UNIQUE INDEX `login_UNIQUE` (`login` ASC) VISIBLE)
+ENGINE = InnoDB
+DEFAULT CHARACTER SET = utf8;
+
 CREATE TABLE `internet_shop`.`products` (
   `product_id` BIGINT(11) NOT NULL AUTO_INCREMENT,
-  `name` VARCHAR(225) NOT NULL,
-  `price` VARCHAR(225) NOT NULL,
+  `productName` VARCHAR(256) NOT NULL,
+  `productPrice` VARCHAR(256) NOT NULL,
   `deleted` TINYINT NULL DEFAULT 0,
-  PRIMARY KEY (`product_id`),
-  UNIQUE INDEX `product_id_UNIQUE` (`product_id` ASC) VISIBLE)
+  PRIMARY KEY (`product_id`))
+ENGINE = InnoDB
+DEFAULT CHARACTER SET = utf8;
+
+CREATE TABLE `internet_shop`.`orders` (
+  `order_id` BIGINT(11) NOT NULL AUTO_INCREMENT,
+  `user_id` BIGINT(11) NOT NULL,
+  PRIMARY KEY (`order_id`))
+ENGINE = InnoDB
+DEFAULT CHARACTER SET = utf8;
+
+CREATE TABLE `internet_shop`.`orders_products` (
+  `product_id` BIGINT(11) NOT NULL,
+  `order_id` BIGINT(11) NOT NULL)
+ENGINE = InnoDB
+DEFAULT CHARACTER SET = utf8;
+
+CREATE TABLE `internet_shop`.`roles` (
+  `role_id` BIGINT(11) NOT NULL AUTO_INCREMENT,
+  `role_name` VARCHAR(256) NOT NULL,
+  PRIMARY KEY (`role_id`),
+  UNIQUE INDEX `role_name_UNIQUE` (`role_name` ASC) VISIBLE)
+ENGINE = InnoDB
+DEFAULT CHARACTER SET = utf8;
+
+CREATE TABLE `internet_shop`.`users_roles` (
+  `user_id` BIGINT(11) NOT NULL,
+  `role_id` BIGINT(11) NOT NULL)
+ENGINE = InnoDB
+DEFAULT CHARACTER SET = utf8;
+
+CREATE TABLE `internet_shop`.`shopping_carts` (
+  `shopping_cart_id` BIGINT(11) NOT NULL,
+  `user_id` BIGINT(11) NOT NULL,
+  PRIMARY KEY (`shopping_cart_id`))
+ENGINE = InnoDB
+DEFAULT CHARACTER SET = utf8;
+
+CREATE TABLE `internet_shop`.`shopping_carts_products` (
+  `shopping_cart_id` BIGINT(11) NOT NULL,
+  `product_id` BIGINT(11) NOT NULL)
 ENGINE = InnoDB
 DEFAULT CHARACTER SET = utf8;
+
```

Comment:
I think you can edit your original SQL script to include everything you added later. So that, when somebody runs this script on another computer, they get the right scheme at once without alterations.

Reply:
Don't really understand what do you mean :(

Reply:
I mean this: now you have two entries `CREATE TABLE `internet_shop`.`shopping_carts_products`` and `ALTER TABLE `internet_shop`.`shopping_carts_products``. I suggest you edit this file so that you only have `CREATE TABLE `internet_shop`.`shopping_carts_products` `